### PR TITLE
docs: update Prisma guide to use Accelerate

### PR DESCRIPTION
**Pull request by ruheni**
Remove mentions of Prisma Data Proxy in favor of Prisma Accelerate.

### CODE REVIEW

File: runtime/tutorials/how_to_with_npm/prisma.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ cd rest-api-with-prisma-oak
 Then, let's run `prisma init` with Deno:
 
 ```shell, ignore
-deno run --allow-read --allow-env --allow-write npm:prisma@^4.5 init
+deno run --allow-read --allow-env --allow-write npm:prisma@latest init
```
Comment (lucacasonato) [inline runtime/tutorials/how_to_with_npm/prisma.md:30]: Please keep this pinned to a recent version.

File: runtime/tutorials/how_to_with_npm/prisma.md [hunk 2]
```diff
@@ -53,39 +53,33 @@ model Dinosaur {
 }
 ```
 
-Prisma should also have generated a `.env` file with `DATABASE_URL`. Let's
-assign `DATABASE_URL` to a PostreSQL connection string. In this example, we'll
+Prisma also have generates a `.env` file with `DATABASE_URL`. Let's
```
Comment (lucacasonato) [inline runtime/tutorials/how_to_with_npm/prisma.md:56]: This sentence doesn't read correctly.

File: runtime/tutorials/how_to_with_npm/prisma.md [hunk 3]
```diff
@@ -53,39 +53,33 @@ model Dinosaur {
...
-remote origin, and push the repo.
+To get started with the Prisma Data Platform:
 
-Next, sign up for a free
-[Prisma Data Platform account](https://cloud.prisma.io/).
+1. Sign up for a free [Prisma Data Platform account](https://console.prisma.io)
+2. Create a project
+3. Navigate to the project you created
+4. Enable Accelerate
+5. Generate an Accelerate connection string and copy it to your clipboard.
```
Comment (lucacasonato) [inline runtime/tutorials/how_to_with_npm/prisma.md:80]: This doesn't mention that you need to enter the original connection string for Postgres anywhere.

File: runtime/tutorials/how_to_with_npm/prisma.md (hunk 1 above)
Comment (ruheni) [reply runtime/tutorials/how_to_with_npm/prisma.md:30]: I recommend sticking to the latest version, which we release every 3 weeks, as it contains the latest features and improvements. We also test every version to ensure it works as it should with Deno and other platforms/ libraries/ frameworks in our [ecosystem-tests repo](https://github.com/prisma/ecosystem-tests).🙂

**Review by kwhinnery — APPROVED** [review]

**Pull request merged**